src/crypto/OutgoingRoomKeyRequestManager.js

```diff
@@ -41,6 +41,9 @@ const ROOM_KEY_REQUEST_STATES = {
 
     /** request sent, awaiting reply */
     SENT: 1,
+
+    /** reply received, cancellation not yet sent */
+    CANCELLATION_PENDING: 2,
```

Comment:
> reply received, cancellation not yet sent

I don't understand what state this represents in the state machine. I understood when it was a simple sent/not sent. Is this accurate:
```
UNSENT -------> SENT
  |              ^
  |              |
  +----> CANCELLATION_PENDING
```
as in, you cannot go to this state if the request has already been sent? It might be nice to have a diagram akin to this somewhere as it helps me grok things. Does it matter that `SENT` can mean "sent cancellation" AND "sent request"?